### PR TITLE
[WFLY-11606]: Upgrade Artemis to 2.6.4

### PR DESCRIPTION
**Pull request by ehsavoie**
* Upgrading Artemis version
* Fixing error message ids

Jira: https://issues.jboss.org/browse/WFLY-11606

### CODE REVIEW

**Comment by ehsavoie** [comment]
@jmesnil could you please review ?

**Comment by kabir** [comment]
@mnovak1 Is this one ok with you?

**Review by jmesnil — APPROVED** [review]

**Comment by mnovak1** [comment]
@kabir We should wait for AMQ QE team to give us ack that Artemis 2.6.4 broker is without regressions. We ran integration tests for Artemis and they passed. So we're waiting for AMQ team atm.

**Comment by bstansberry** [comment]
@dosoudil FYI. Given the above I don't know if this will get in 16.0.0.Beta1 or not. It's just a micro, I wanted to point it out.

**Comment by kabir** [comment]
Adding the hold label so we don't accidentally merge until we hear from AMQ QE

**Comment by kabir** [comment]
@ehsavoie before merging this we need to make sure that the dependency tree is the same, or adjust for any exclusions. We've had some problems with artemis upgrades in the past and only noticing in the product branch which has some stricter checks

CC @psakar @dosoudil

File: pom.xml [hunk 1]
```diff
@@ -266,7 +266,7 @@
         <version.jsoup>1.8.3</version.jsoup>
         <version.net.jcip>1.0</version.net.jcip>
         <version.net.shibboleth.utilities.java-support>7.1.1</version.net.shibboleth.utilities.java-support>
-        <version.org.apache.activemq.artemis>2.6.3.jbossorg-00014</version.org.apache.activemq.artemis>
+        <version.org.apache.activemq.artemis>2.6.4</version.org.apache.activemq.artemis>
```
Comment (bstansberry) [inline pom.xml]: Does an upgrade of org.apache.qpid:proton-j come with this?

Comment (ehsavoie) [reply pom.xml]: no, there is no dependency change (no netty, nor proton-j)

**Comment by ehsavoie** [comment]
This should be replaced by an upgrade to Apache Artemis 2.7.0 to be released soon

**Pull request closed without merging**